pkg/reconciler/v1alpha1/route/table_test.go

```diff
@@ -1552,7 +1552,7 @@ func patchRemoveLabel(namespace, name, key string) clientgotesting.PatchActionIm
 	action.Name = name
 	action.Namespace = namespace
 
-	patch := fmt.Sprintf(`{"metadata":{"labels":{"%s":null}}}`, key)
+	patch := fmt.Sprintf(`{"metadata":{"labels":{"%s":null},"resourceVersion":""}}`, key)
```

Comment:
It's worth updating the fixture (ie. the configuration) with a resourceVersion and modifying these functions to accept a resourceVersion param.

We want to assert that the correct patches are being created when the resourceVersion is set.